transport-native-epoll/src/main/c/io_netty_channel_unix_FileDescriptor.c

```diff
@@ -24,3 +25,16 @@ JNIEXPORT int JNICALL Java_io_netty_channel_unix_FileDescriptor_close(JNIEnv* en
    }
    return 0;
 }
+
+JNIEXPORT int JNICALL Java_io_netty_channel_unix_FileDescriptor_open(JNIEnv* env, jclass clazz, jstring path) {
+
+    const char* f_path = (*env)->GetStringUTFChars(env, path, 0);
+
+    int res = open(f_path, O_WRONLY | O_CREAT | O_TRUNC, 0644);
```

Comment:
@trustin after thinking more about it I think 0644 is good as if the user has set an umask it will kick in and may "tighten" stuff up. Or I'm missing something ?